CMake/FindCFITSIO.cmake

```diff
@@ -32,6 +32,10 @@
 
 if(NOT CFITSIO_FOUND)
 
+  if(NOT CFITSIO_ROOT_DIR AND DEFINED ENV{CFITSIO_ROOT_DIR})
+    set(CFITSIO_ROOT_DIR "$ENV{CFITSIO_ROOT_DIR}")
+  endif()
```

Comment:
DAS5 defines a nice CFITSIO_ROOT_DIR environment variable which was unused until now: We manually had to add -DCFITSIO_ROOT_DIR=... to the cmake invocation. With this change, that's no longer necessary.